unlock-protocol-com/blog/airdrop-nft-memberships.md

```diff
@@ -0,0 +1,58 @@
+---
+
+Creators can easily deploy their membership contracts (called Locks) using the Unlock dashboard. Each Lock has different membership terms: membership price, number of members, duration of the membership (yes, the NFT does expire!)... etc.
+
+Memberships can then be purchased directly by the fans. This happens by calling the [`purchase` function](https://docs.unlock-protocol.com/developers/smart-contracts/lock-api#purchase) on the Lock contract. In practice, this function can actually be called with another recipient's address, in order for them to receive the NFT (instead of the buyer).
+
+However, in a lot of cases, creators may want to "give away" or **airdrop NFT memberships** for free to some of their users. Here are examples of why this is useful:
+
+- This is how our [credit card](/blog/credit-card-nft) flow works: the NFT is paid "off chain" via a credit card transaction, and _granted_ on chain (technically, no on-chain payment happens).
+- Someone might want to give a "limited time" trial. For example, my [personal blog](https://ouvre-boite.com) has a membership of its own, where NFTs expire after 1 year, but where anyone who [follows me on Twitter](https://twitter.com/julien51) [can claim a 30 minute](https://claim-ouvre-boite-membership.herokuapp.com/) trial!
+- Finally, some communities and memberships cannot be purchased, but only be granted. That is the case for the [PlannerDAO certifications](https://twitter.com/PlannerDAO/status/1479097169747529735).
+
+# How to airdrop NFT memberships
+
+## Using the Dashboard
+
+Once your lock is deployed, you can head out to the members page.
+
+![Dashboard](/images/blog/airdrop-nft-memberships/dashboard.png)
+
+On that page you will see the list of all the members on your lock. You can also see a button that lets you airdrop an NFT: click on it!
+
+![Members Page](/images/blog/airdrop-nft-memberships/members-page.png)
+
+A drawer will open and let you easily airdrop a membership to any address.
+
+- Make sure the lock is the correct one!
+
+- You need to enter the recipient's address. The field supports ENS names, in order to avoid making typos!
+
+- Next you can select when this airdropped membership is going to expire. By default, it is based on the duration of the lock, but can be customized. This flexibility means you can grant a key for a few minutes for trials!
+
+- Finally you can select a "key manager". At Unlock, we decouple the "owner" of an NFT from the account who has the ability to transfer said NFT. Most of the time, and by default, it is the same... but in some cases, it can be important to make sure that someone, even if they own an NFT is not able to transfer it. That's the case for credit card purchases. For security reasons, we want to make sure that NFT "bought" this way are not transferable. Similarly, granted keys can be "cancelled" by their owner, triggering a refund, by default. So it can be important to retain the key manager role.
```

Comment:
Last, but never least, you may delegate a "key manager". At Unlock, we decouple the "owner" of an NFT from the account who possess transfer abilities for the NFT in question. By default and in many cases, the owner and manager are the same. But certain unique use-cases do require the blocking of transfer access by the NFT owner. For security reasons, we want to ensure that the NFT "bought" this way are non-transferable. Similarly, granted keys can be "cancelled" by their owner, and by default, triggering a refund. In some cases, retaining the key manager role is important.